### PR TITLE
Update raku kernel to reflect name change

### PR DESCRIPTION
**Pull request by sumanstats**
Recently the community has undergone name change from Perl 6 --> Raku.

Here are the recommendations for file extensions https://github.com/Raku/problem-solving/blob/master/solutions/language/Path-to-Raku.md#extensions

Since rakudo release 2019.11, `raku` interpreter is available to execute raku scripts.

Example: `raku script.raku`
So to reflect this I have edited the files with new extensions, updated readme. I installed and tested in local linux machine with `rakudo 2019.11` and it indeed works.
May be name change of the repo would make it uniform with the raku changes.

Thanks.

### CODE REVIEW

**Comment by bduggan** [comment]
Thanks for the work!  I think there is a bit more to do (for travis testing, and somehow specifying a minimum version of the compiler) but this is a great start.

**Comment by sumanstats** [comment]
With modification in `travis.yml`, CI is also working now.
Matrix expansion will enable testing in future releases of rakudo.

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-  - travis_retry zef install --depsonly .
-sudo: false
+  matrix:
+    - RAKUDO_VERSION=daily
+    - RAKUDO_VERSION=2019.11
+    
+ 
+
+script: |
+  docker run -it -v $(pwd):/home -w /home sumankhanal/rakudo:${RAKUDO_VERSION} /bin/sh -c "
```
Comment (bduggan) [inline .travis.yml]: I'd rather not introduce this dependency

File: bin/jupyter-kernel.raku [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl6
+#!/usr/bin/env raku
```
Comment (bduggan) [inline bin/jupyter-kernel.raku]: Is a `raku` executable available for os/x without building from source?

**Comment by bduggan** [comment]
I will probably wait until it is possible to `brew install raku` on os/x before merging a change to the shebang line, and I'll wait for more editor support before changing the file extensions

File: bin/jupyter-kernel.raku (hunk 2 above)
Comment (tinmarino) [reply bin/jupyter-kernel.raku]: Not on my Ubuntu 19.10

**Comment by bduggan** [comment]
I've opened #79 to address the conflicts and finish this up.

thanks for all the work!

**Comment by bduggan** [comment]
I've merged #79 so I'll go ahead and close this one.
